### PR TITLE
feat: audit logs support added

### PR DESCRIPTION
**Pull request by 1fawadshah**
Feature: GCP Audit Logs support has been added to the module.

### CODE REVIEW

**Comment by 1fawadshah** [comment]
Docs to be generated

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -217,6 +217,79 @@ section {
...
+                configs = [
+                  {
+                    log_type = "DATA_READ"
+                  },
+                  {
+                    log_type = "DATA_WRITE"
+                  },
+                  {
+                    log_type = "ADMIN_READ"
+                  }
```
Comment (soerenmartius) [inline README.tfdoc.hcl:239]:
```suggestion
                  {
                    log_type = "ADMIN_READ"
                  },
                  {
                    log_type = "ADMIN_WRITE"
                  },
```

Let's add a full example here

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Thanks for contributing! Could you add the changes to the `CHANGELOG.md` also?

Many thanks

File: README.tfdoc.hcl (hunk 1 above)
Comment (1fawadshah) [reply README.tfdoc.hcl:239]: ADMIN_WRITE is not a valid log_type
Audit Logs have 3 valid log types; ADMIN_READ, DATA_READ and DATA_WRITE
https://cloud.google.com/logging/docs/audit/configure-data-access#configuration_overview

**Comment by 1fawadshah** [comment]
> Thanks for contributing! Could you add the changes to the `CHANGELOG.md` also?
> 
> Many thanks

Added !

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -217,6 +217,79 @@ section {
             }
           }
         }
+
+        variable "audit_configs" {
+          type = object(audit_log)
```
Comment (mariux) [inline README.tfdoc.hcl:222]:
```suggestion
          type = object(audit_config)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.3]
+
```
Comment (mariux) [inline CHANGELOG.md:11]: this needs to have links adjusted at the end of the file to work. please adjust or remove the version tag here

```suggestion
```

**Review by mariux — DISMISSED** [review]
thanks for the contribution.. minor changes required.

File: README.tfdoc.hcl (hunk 2 above)
Comment (1fawadshah) [reply README.tfdoc.hcl]: Change made and requested Nathan to regenerate docs

File: CHANGELOG.md [hunk 4]
```diff
@@ -21,5 +26,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Initial Implementation
 
 [unreleased]: https://github.com/mineiros-io/terraform-google-organization-iam/compare/v0.0.2...HEAD
```
Comment (soerenmartius) [inline CHANGELOG.md:28]:
```suggestion
[unreleased]: https://github.com/mineiros-io/terraform-google-organization-iam/compare/v0.0.3...HEAD
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Thanks @1fawadshah , just one last small change before we can merge this!

File: CHANGELOG.md (hunk 4 above)
Comment (1fawadshah) [reply CHANGELOG.md]: Ahh missed it, updated now !

**Pull request merged**